lib/hyrax/controlled_vocabularies/location_decorator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module LocationDecorator
```

Comment:
@summer-cook based on the file path and original class name, shouldn't this decorator be: 

```ruby
Hyrax::ControlledVocabularies::LocationDecorator
# instead of just 
Hyrax::LocationDecorator
```

Reply:
so it would be like 

```
module Hyrax
    module ControlledVocabularies
        module LocationDecorator
```

?

i'm going to try it and let you know if it works

Reply:
@bkiahstroud ok so both ways work somehow. It seems like the second way is a bit more clear, so will push that up